config/istio/istiod/configmap.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+data:
+  mesh: |-
+    defaultConfig:
+      discoveryAddress: istiod.istio-system.svc:15012
```

Comment:
This line is the only change from the default configmap bundled with the chart.
It replaces `discoveryAddress: istiod.istio.svc:15012` with `discoveryAddress: istiod.istio-system.svc:15012`